package.json

```diff
@@ -10,23 +10,26 @@
   "scripts": {
     "postinstall": "npm ls --depth=0",
     "start": "gulp dev",
-    "heroku": "gulp copy-assets && gulp sassdoc && node app/start.js",
+    "heroku": "npm run build:assets && node app/start.js",
     "build-release": "./bin/build-release.sh",
     "publish-release": "./bin/publish-release.sh",
     "pre-release": "./bin/pre-release.sh",
+    "build:assets": "gulp copy-assets && gulp sassdoc",
```

Comment:
Should this run `test:build:assets` similar to what the other two build tasks do?

Reply:
Oh, would that cause an infinite loop because of `pretest`? 🤔

Reply:
This might be irrelevant based on https://github.com/alphagov/govuk-frontend/pull/2850#discussion_r973188316

Reply:
Yeah think so, thanks

This line was a precursor to https://github.com/alphagov/govuk-frontend/issues/2855 so any more improvements can come after this PR if needed

Reply:
I'll mark this one as resolved (answered on https://github.com/alphagov/govuk-frontend/issues/2855)